models/surgical_tool_segmentation/docs/readme.md

```diff
@@ -0,0 +1,95 @@
+# Description
+A pre-trained model for the surgical tool segmentation task.
+
+# Model Overview
+This model is trained using a flexible unet structure with an efficient-b0 [1] as the backbone and a UNet architecture [2] as the decoder. Datasets use private samples from [Activ Surgical](https://www.activsurgical.com/).
+The [pytorch model](https://drive.google.com/file/d/14r6WmzaZrgaWLGu0O9vSAzdeIGVFQ3cs/view?usp=sharing) and [torchscript model](https://drive.google.com/file/d/1i-e5xXHtmvmqitwUP8Q3JqvnmN3mlrEm/view?usp=sharing) are shared in google drive. Details can be found in large_files.yml file.
+
+## Data
+Datasets used in this work were provided by [Activ Surgical](https://www.activsurgical.com/).
+
+Since datasets are private, existing public datasets like [EndoVis 2017](https://endovissub2017-roboticinstrumentsegmentation.grand-challenge.org/Data/) can be used to train a similar model.
+
```

Comment:
Can we add a reference to what needs to be changed in configs/inference.json when users bring their own data? For example, "dataset_dir", "datalist" need to change, any more?

Reply:
Hi @jin-nvidia ,
     Thank you for your advice. A description of what need to do with user dataset was added to the readme file.
Thanks,
Bin